[2022-01-30 01:27:45,050] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_fhv.download_dataset_task manual__2022-01-30T01:26:19.408088+00:00 [queued]>
[2022-01-30 01:27:45,067] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_fhv.download_dataset_task manual__2022-01-30T01:26:19.408088+00:00 [queued]>
[2022-01-30 01:27:45,068] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-30 01:27:45,068] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-01-30 01:27:45,068] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-30 01:27:45,098] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-01-30 01:26:19.408088+00:00
[2022-01-30 01:27:45,110] {standard_task_runner.py:52} INFO - Started process 362 to run task
[2022-01-30 01:27:45,118] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_fhv', 'download_dataset_task', 'manual__2022-01-30T01:26:19.408088+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_fhv.py', '--cfg-path', '/tmp/tmp6d1vqvt5', '--error-file', '/tmp/tmp2u57odlb']
[2022-01-30 01:27:45,119] {standard_task_runner.py:77} INFO - Job 31: Subtask download_dataset_task
[2022-01-30 01:27:45,233] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_fhv.download_dataset_task manual__2022-01-30T01:26:19.408088+00:00 [running]> on host 04ca2676bdcc
[2022-01-30 01:27:45,358] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_fhv
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-30T01:26:19.408088+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-30T01:26:19.408088+00:00
[2022-01-30 01:27:45,360] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-30 01:27:45,361] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSLf https://nyc-tlc.s3.amazonaws.com/trip+data/fhv_tripdata_2019-01.csv > /opt/***/fhv_tripdata_2019-01.csv']
[2022-01-30 01:27:45,386] {subprocess.py:85} INFO - Output:
[2022-01-30 01:29:01,473] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-30 01:29:02,536] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_fhv, task_id=download_dataset_task, execution_date=20220130T012619, start_date=20220130T012745, end_date=20220130T012902
[2022-01-30 01:29:02,568] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-30 01:29:02,611] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
